server/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dmgt-tech/the-usher-server",
-  "version": "1.6.1",
+  "version": "2.0.0",
   "description": "The Usher Authorization Server",
   "engines": {
     "node": ">=16"
```

Comment:
@k1ch  do you want to bump this to 18 since we are just considering LTS versions at this point? Thanks